tests/build_tests/super_errors/input.js

```diff
@@ -10,8 +10,8 @@ const { bsc } = setup(import.meta.dirname);
 
 const expectedDir = path.join(import.meta.dirname, "expected");
 
-const fixtures = readdirSync("fixtures").filter(
-  fileName => path.extname(fileName) === ".res",
+const fixtures = readdirSync(path.join(import.meta.dirname, "fixtures")).filter(
+  (fileName) => path.extname(fileName) === ".res"
```

Comment:
Why did the formatting change here?

Reply:
You mean the parenthesis on `(fileName)`?